docs/migration-guides/prometheus-migration.md

```diff
@@ -0,0 +1,328 @@
+# Prometheus Time Series Database migration
```

Comment:
Migration - I would "word case" the doc title

Reply:
done.

Reply:
done.